public/pages/SplitIndex/container/SplitIndex/SplitIndex.tsx

```diff
@@ -43,6 +44,7 @@ export class SplitIndex extends Component<SplitIndexProps> {
   };
 
   async componentDidMount() {
+    this.context.chrome.setBreadcrumbs([BREADCRUMBS.INDEX_MANAGEMENT, BREADCRUMBS.INDICES, BREADCRUMBS.SPLIT_INDEX]);
```

Comment:
The last path will be a dynamic breadcrumb, it's recommended to add current path with dynamic params as the last breadcrumb, which should be { ...BREADCRUMBS.SPLIT_INDEX, path: '${your_path_and_queries_to_split_index}' }

Reply:
Updated in latest code